cpp/src/arrow/dataset/filter_test.cc

```diff
@@ -62,7 +62,13 @@ class ExpressionsTest : public ::testing::Test {
     auto simplified = expr.Assume(given);
     ASSERT_EQ(E{simplified}, E{expected})
         << "  simplification of: " << expr.ToString() << std::endl
-        << "              given: " << given.ToString() << std::endl;
+        << "              given: " << given.ToString() << std::endl
+        << "           expected: " << expected.ToString() << std::endl;
```

Comment:
This shouldn't be necessary; GTest should print the expected and actual values automatically